eahub/templates/eahub/profile.html

```diff
@@ -1,295 +1,302 @@
-{% extends 'base.html' %}
+{% extends 'base_bs5.html' %}
+
+{% load thumbnail crispy_forms_tags sekizai_tags static feature_flags %}
 
-{% load thumbnail %}
-{% load static %}
-{% load crispy_forms_tags %}
 
 {% block title %}{{ profile.name }}{% endblock %}
 
+{% block navbar %}
+    {% include 'components/navbar.html' %}
+{% endblock %}
+
 {% block content %}
+    {% if profile.image %}
+        {% block meta_image %}
+            <meta property="og:image" content="{% thumbnail profile.image "524x274" crop="top" %}">
+            <meta property="og:image:width" content="524">
+            <meta property="og:image:height" content="274">
+            <meta property="og:image:type" content="image/png">
+        {% endblock %}
+    {% endif %}
+    {% block meta_info %}
+        <meta name="description" content="{{ profile.summary }}">
+        <meta property="og:type" content="profile">
+        <meta property="og:title" content="{{ profile.name }} - EA Hub">
+        <meta property="og:description" content="{{ profile.summary }}">
+    {% endblock %}
 
-<br><br>
+    {% addtoblock 'body_classes' %}profile-detail{% endaddtoblock %}
 
-{% if messages %}
-    <div class="container">
+    {% include 'includes/messages.html' %}
+
+    <div class="container prof__header mt-6 mb-5">
         <div class="row">
-            <div class="col-md-4 col-md-offset-4">
-                {% for message in messages %}
-                    <div {% if message.tags %}
-                        class="alert alert-{{ message.tags }} hub-{{ message.tags }} alert alert-dismissable"{% endif %}>
-                        <button type="button" class="close" data-dismiss="alert"
-                            aria-label="Close"
-                        >
-                            <span aria-hidden="true">&times;</span>
-                        </button>
-                        <div>{{ message }}</div>
+            <div class="col-xs-24 col-md-auto mb-3 mb-lg-0">
+                {% thumbnail profile.image "200x200" crop="center" as im %}
```

Comment:
Navbar seems to cut off pic and linkedin logo  
![image](https://user-images.githubusercontent.com/15908208/108321532-38764f00-71bc-11eb-9406-0e0f6c998241.png)

Reply:
looks like your browser is using cache and styles aren't present - try a hard reload